spec/services/invitation/insert_from_invite_spec.rb

```diff
@@ -22,6 +22,34 @@ def existing_user_invite
       email:         @user.email)
   end
 
+  describe '#execute' do
+    context 'inviting network partner as participant' do
+      let!(:existing_user) { FactoryGirl.create(:user, :with_network_partner_role) }
```

Comment:
If you're using what you declare in a `let` block later on, you don't need the `!`; `let` is lazy and `let!` is eager.